### PR TITLE
New design

### PR DESCRIPTION
**Pull request by NNieuwenhuis**
New Design

### CODE REVIEW

File: .angular-cli.json [hunk 1]
```diff
@@ -20,6 +20,8 @@
       "prefix": "app",
       "styles": [
         "../node_modules/bootstrap/dist/css/bootstrap.css",
+        "../node_modules/angular-bootstrap-md/scss/bootstrap/bootstrap.scss",
```
Comment (lukeelten) [inline .angular-cli.json]: Bootstrap is already included (see line above).
Please only include bootstrap ONCE!! Otherwise this will slow down the browser.

**Review by Lehren — APPROVED** [review]
Noice

File: .angular-cli.json (hunk 1 above)
Comment (Lehren) [reply .angular-cli.json]: Isn't this just the material design addon for bootstrap? I'm not sure.

Comment (lukeelten) [reply .angular-cli.json]: I was as well and I looked it up. bootstrap.scss contains the whole bootstrap library.

**Pull request merged**